var/spack/repos/builtin/packages/sperr/package.py

```diff
@@ -12,21 +12,20 @@ class Sperr(CMakePackage):
 
     # Package info
     homepage = "https://github.com/NCAR/SPERR"
-    url = "https://github.com/NCAR/SPERR/archive/refs/tags/v0.6.2.tar.gz"
+    url = "https://github.com/NCAR/SPERR/archive/refs/tags/v0.7.1.tar.gz"
     git = "https://github.com/NCAR/SPERR.git"
     maintainers("shaomeng", "robertu94")
 
     # Versions
     version("main", branch="main")
+    version("0.7.1", sha256="1c3f46200be365427d1f57f5873f1b0b6dbcd297de4603a47a7fa3f41b273d79")
     version("0.6.2", sha256="d986997e2d79a1f27146ad02c623359976a1e72a1ab0d957e128d430cda3782d")
     version("0.5", sha256="20ad48c0e7599d3e5866e024d0c49648eb817f72ad5459f5468122cf14a97171")
 
     depends_on("git", type="build")
-    depends_on("zstd", type=("build", "link"), when="+zstd")
     depends_on("pkgconfig", type=("build"), when="+zstd")
```

Comment:
Does the dependency on `pkgconfig` also need to be removed?  If not, then the `when` argument needs to be removed.

Reply:
It's a good point! I removed the "when" argument!